spec/features/idv/steps/gpo_otp_verification_step_spec.rb

```diff
@@ -44,7 +33,14 @@
 
   context 'ThreatMetrix disabled, but we have ThreatMetrix status on proofing component' do
     let(:threatmetrix_enabled) { false }
-    let(:fraud_pending_reason) { 'threatmetrix_review' }
+    let(:profile) do
+      create(
+        :profile,
+        :verify_by_mail_pending,
+        :with_pii,
+        :fraud_review_pending,
+      )
+    end
```

Comment:
This is an invalid profile state. A user should not be fraud_review_pending while they are still verify_by_mail_pending.